lib/interactor.rb

```diff
@@ -1,6 +1,9 @@
 require "interactor/context"
 require "interactor/error"
 require "interactor/hooks"
+require "interactor/contract/property"
+require "interactor/contract/property_table"
+require "interactor/contract/contract"
```

Comment:
I think the only `require` that's needed is for `Contract`.

Also, why have `interactor/contract/contract` instead of just `interactor/contract`. Seems like that would be the better home for it.